src/include/cache.h

```diff
@@ -290,6 +294,11 @@ struct __wt_cache_pool {
     uint8_t flags;
 };
 
+/*
+ * Optimize comparisons against the lookaside URI, flag handles that reference the lookaside file.
+ */
+#define WT_IS_BTREE_LOOKASIDE(session) F_ISSET(S2BT((session)), WT_BTREE_LOOKASIDE)
```

Comment:
Sorry to be a pedant, but it's actually the DHANDLE not the BTREE. I'd call this `WT_DHANDLE_IS_LOOKASIDE`

Reply:
Corrected.